src/LinkDotNet.Blog.Domain/Introduction.cs

```diff
@@ -2,9 +2,11 @@
 
 public sealed record Introduction
 {
+    public const string IntroductionSection = "Introduction";
```

Comment:
We could use `nameof(Introduction)` instead of the hard string. What's your thought on that one?

Reply:
I think it will make configuration section tightly coupled with record name, it is okay for Introduction but for Authentication, Giscus and others we will have to change either record or configuration section name.